server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala

```diff
@@ -501,6 +501,76 @@ trait WebSocketContract {
       .contains(mailboxStateChange, emailStateChange)
   }
 
+  @Test
+  // For client compatibility purposes
+  def specifiedUnHandledDataTypesShouldNotBeRejected(server: GuiceJamesServer): Unit = {
+    val bobPath = MailboxPath.inbox(BOB)
+    val accountId: AccountId = AccountId.fromUsername(BOB)
+    val mailboxId = server.getProbe(classOf[MailboxProbeImpl]).createMailbox(bobPath)
+
+    Thread.sleep(100)
+
+    val response: Either[String, List[String]] =
+      authenticatedRequest(server)
+        .response(asWebSocket[Identity, List[String]] {
+          ws =>
+            ws.send(WebSocketFrame.text(
+              """{
+                |  "@type": "WebSocketPushEnable",
+                |  "dataTypes": ["Mailbox", "Email", "VacationResponse", "Thread", "Identity", "EmailSubmission", "EmailDelivery"]
```

Comment:
Missing `SearchSnippet` from the mail spec?

Reply:
No /changes on this one, it do not make sense either to request push for it as a SearchSnipset is only bound to a specific "search".

This object has no state property and is not something you want to synchronize.

The spec is unclear on that.

Reply:
ah yeah you are right no /changes for it my bad!